src/front-door/azext_front_door/_params.py

```diff
@@ -129,15 +129,17 @@ def load_arguments(self, _):
     with self.argument_context('network front-door routing-rule', arg_group=None) as c:
         c.argument('accepted_protocols', nargs='+', help='Space-separated list of protocols to accept. Default: Http')
         c.argument('patterns_to_match', options_list='--patterns', nargs='+', help='Space-separated list of patterns to match. Default: \'/*\'.')
-        c.argument('rules_engine', help='Name or ID of a Rules Engine configuration.', validator=validate_rules_engine)
+        c.argument('rules_engine', help='Name or ID of a Rules Engine configuration. To unlink property, \"--remove rulesEngine\"', validator=validate_rules_engine)
         c.argument('frontend_endpoints', help='Space-separated list of frontend endpoint names or IDs.', nargs='+', validator=validate_frontend_endpoints)
     with self.argument_context('network front-door routing-rule', arg_group='Forward Routing Rule') as c:
         c.argument('backend_pool', help="Name or ID of a backend pool. It's required to create a Forward routing rule.", validator=validate_backend_pool)
-        c.argument('forwarding_protocol', help='Protocol to use for forwarding traffic.')
+        c.argument('forwarding_protocol', arg_type=get_enum_type(FrontDoorForwardingProtocol), help='Protocol to use for forwarding traffic.')
```

Comment:
Extract this argument to a CLIArgumentType for reuse as there are multiple places use this argument.

Reply:
I will consider this in the next commit, for now let's get this merged as it is just a make it "pretty"change.